packages/logrotate/build

```diff
@@ -21,9 +21,9 @@ nomail
 
 /var/lib/dcos/mesos/log/* {
     olddir /var/lib/dcos/mesos/log/archive
-    maxsize 2000k
-    daily
-    rotate 7
+    maxsize 256M
```

Comment:
One of the advantages of the systemd-journald approach is they actually will cap at a percentage of the underlying disk backing the journal, so that this sort of thing is automatically taken care of / they guarantee that the disk won't fill up because of something the jounral wrote.